generate.sh

```diff
@@ -103,11 +159,17 @@ function ExtractABI {
 function InvokeProjectionTooling {
     local framework=$1
 
-    $scriptroot/dotnet.sh $scriptroot/artifacts/bin/Swift.Bindings/$configuration/net9.0/Swift.Bindings.dll -a "./$framework.abi.json" -d "/System/Library/Frameworks/$framework.framework/$framework" -t "/Applications/Xcode.app/Contents/Developer/Platforms/$platform.platform/Developer/SDKs/$platform.sdk/System/Library/Frameworks/$framework.framework/$framework.tbd" -o "./"
+    if $tool; then
+        echo "Using tool to generate bindings for framework '$framework'"
+        $scriptroot/dotnet.sh swiftbindings -a "./$framework.abi.json" -d "/System/Library/Frameworks/$framework.framework/$framework" -t "/Applications/Xcode.app/Contents/Developer/Platforms/$platform.platform/Developer/SDKs/$platform.sdk/System/Library/Frameworks/$framework.framework/$framework.tbd" -o "./"
```

Comment:
I'm not sure this will work correctly for MacCatalyst... 

Looking at the `StoreKit.tbd`, I see that `MacOSX.platform` directory has a subdirectory `iOSSupport` which I believe is for `MacCatalyst`. `./Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/iOSSupport/System/Library/Frameworks/StoreKit.framework/StoreKit.tbd`

Reply:
Good catch, updated.